packages/commons/src/config/index.ts

```diff
@@ -1,12 +1,39 @@
 import { z } from "zod";
+import { APIEndpoint } from "./../model/apiEndpoint.js";
 
-const Config = z
+const JWTConfig = z
+  .discriminatedUnion("SKIP_JWT_VERIFICATION", [
+    z.object({
+      SKIP_JWT_VERIFICATION: z.literal("true"),
+    }),
+    z.object({
+      SKIP_JWT_VERIFICATION: z.literal("false"),
+      WELL_KNOWN_URLS: z
+        .string()
+        .transform((s) => s.split(",").map((s) => APIEndpoint.parse(s))),
+    }),
+  ])
+  .transform((c) =>
+    c.SKIP_JWT_VERIFICATION === "false"
+      ? {
+          skipJWTVerification: false as const,
+          wellKnownUrls: c.WELL_KNOWN_URLS,
+        }
+      : {
+          skipJWTVerification: true as const,
+        }
+  );
```

Comment:
Adding the JWT config here inside the `commons` model simplifies the well-known clients initialization but requires that each service that uses the `commons` config defines these variables (at least `SKIP_JWT_VERIFICATION=true`)

Maybe we should extract these variables from the `common` package or change how we handle the config initialization.